templates/app-autoscaler-deployment.yml

```diff
@@ -14,25 +14,22 @@ releases:
   url: https://bosh.io/d/github.com/cloudfoundry/bosh-dns-aliases-release?v=0.0.3
   version: '0.0.3'
   sha1: b0d0a0350ed87f1ded58b2ebb469acea0e026ccc
-- name: routing
-  url: https://bosh.io/d/github.com/cloudfoundry-incubator/cf-routing-release?v=0.162.0
-  sha1: f17cf09d2414f5f486d18bbd57b17fd48fb69773
-  version: '0.162.0'
+- name: "routing"
+  version: "0.221.0"
+  url: "https://bosh.io/d/github.com/cloudfoundry/routing-release?v=0.221.0"
+  sha1: "accad1fe9072e96477d3632dc8584e30c243c3ba"`
```

Comment:
@silvestre this contains a backtick that I'd previously removed

Reply:
Hah, shouldn't attempt to commit while talking, I guess :sweat_smile: